codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateClientExtensionHandshakerTest.java

```diff
@@ -90,7 +90,7 @@ public void testCustomHandshake() {
 
         parameters = new HashMap<String, String>();
         parameters.put(CLIENT_MAX_WINDOW, "12");
-        parameters.put(SERVER_MAX_WINDOW, "10");
+        parameters.put(SERVER_MAX_WINDOW, "8");
```

Comment:
@ktqco did this change have the tests fail before your fix ?

Reply:
Yes. :)